### PR TITLE
Alert avatar badges

### PR DESCRIPTION
**Pull request by archit7448**
updated alert , badges and button documentation structure
https://agitated-hawking-b2d497.netlify.app/components/alert/alert

### CODE REVIEW

File: Main/Main.css [hunk 1]
```diff
@@ -6,17 +6,21 @@
     border: 0px;
 }
 :root {
-    --color-primary-green : #2ecc71;
+    --color-primary-blue : #1F51F1   ;
```
Comment (purvasharmadev) [inline Main/Main.css]: Avoid using color names in variables.

File: Components/Badges/badges.html [hunk 2]
```diff
@@ -1,47 +1,143 @@
...
+            <li><a href="/Components/input/input.html">INPUT</a></li>
+            <li><a href="/Components/list/list.html">LIST</a></li>
+            <li><a href="/Components/modal/modal.html">MODAL</a></li>
+            <li><a href="/Components/text-utilities/text-utilities.html">TEXT-UTILITIES</a></li>            
+            <li><a href="/Components/navigation/navigation.html">NAVIGATION</a></li>
+            <li><a href="/Components/ratings/ratings.html">RATINGS</a></li>
+            <li><a href="/Components/slider/slider.html">SLIDER</a></li>              
+          </ul>
+        </nav>
+        <div class="sidebar-wrapper">
```
Comment (amartya-rs) [inline Components/Badges/badges.html]: You can use semantic html like aside or nav for the sidebar.

File: Main/Main.css (hunk 1 above)
Comment (archit7448) [reply Main/Main.css]: it is easy for me to remember

File: Components/Badges/badges.html (hunk 2 above)
Comment (archit7448) [reply Components/Badges/badges.html]: I have updated it

Comment (amartya-rs) [reply Components/Badges/badges.html]: Cool, you can merge Archit.

**Pull request merged**